master/Makefile

```diff
@@ -1,7 +1,7 @@
 .DEFAULT_GOAL := build
 SHELL := bash
 GENERATED := packaging/LICENSE
-GENERATION_INPUTS = ../schemas/gen.py $(shell find ./pkg/schemas/ -name '*.go') $(shell find ../schemas/expconf -name '*.json')
```

Comment:
Isn't this just dropping inputs from being listed?  But the go files are still inputs to the code generation, so it seems like this change is wrongly dropping them.

Reply:
Yes, it's dropping inputs from being listed, but those inputs don't exist. The `zgen_*.go` files that it's complaining it can't find are generated by `../schemas/gen.py`, not any .go files.

Reply:
@rb-determined-ai i recommended this solution. after looking at the code, i was under the impression (and it lined up with memory) that `./pkg/schemas/ -name '*.go'` is the generation output, not input.

what is happening is, as far as i know make, when you run `make -C master clean build` and it calculates `GENERATION_INPUTS` once, including generated code generated by `build/schema_gen.stamp` in `./pkg/schemas` and then `build/schema_gen.stamp` finds it is missing these inputs and implodes.

it didn't happen if you ran `make -C master clean && make -C master build` because `GENERATION_INPUTS` gets computed twice and in `build` the files `find` would return are already rm'd.

Reply:
i am ready to approve this code, but since @rb-determined-ai wrote it, i'll hold for him incase i am missing something.

Reply:
`//go:gen` comments appear in go code, and generate output go code (the zgen stuff).

Example, `SlurmConfigV0`'s `//go:gen` comment results in `zgen_slurm_config_v0.go`:

```go
// SlurmConfigV0 configures experiment resource usage.
//
//go:generate ../gen.sh
type SlurmConfigV0 struct {
	RawSlotsPerNode *int     `json:"slots_per_node,omitempty"`
	RawGpuType      *string  `json:"gpu_type,omitempty"`
	RawSbatchArgs   []string `json:"sbatch_args,omitempty"`
}
```

So if what you need to do is exclude `zgen_*` files from GENERATION_INPUTS, right?

What about this:

```bash
$(shell find ./pkg/schemas/ -name 'zgen_*.go' -prune -o -name '*.go' -print)
```

Reply:
Gosh `find` has the worst command-line interface of any tool I know.